package.json

```diff
@@ -1,6 +1,16 @@
 {
   "name": "strong-mesh-client",
   "version": "2.0.1",
+  "keywords": [
+    "strongloop",
+    "mesh"
+  ],
+  "homepage": "https://github.com/strongloop/strong-mesh-client",
+  "bugs": {
+    "url": "https://github.com/strongloop/strong-mesh-client/issues"
+  },
+  "author": "StrongLoop <callback@strongloop.com>",
+  "readmeFilename": "README.md",
```

Comment:
I thought that was default for some reason, do I need to start adding this field?